project/admin/utils.py

```diff
@@ -49,6 +49,7 @@ def get(self, entry_id):
             entry=entry)
 
     def post(self, entry_id):
+        entry = None
```

Comment:
This variable isn't used in 
`if entry_id is None:`
case. Why do you create this here?